Jenkinsfile

```diff
@@ -86,7 +86,7 @@ pipeline {
                defaultValue: '',
                description: 'Package version to use instead of building. example: 1.3.103-1, 1.2-2')
         string(name: 'CI_HARDWARE_DISTRO',
-               defaultValue: 'centos7',
+               defaultValue: '',
```

Comment:
should this be centos8?

Reply:
No.  We really don't want default values for parameters due to the ugly way they work with Jenkins pipelines.  They take two runs to alter their value.